build/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+HOST_DIR=$(cd $(dirname "$0")/../; pwd)
+CONTAINER_DIR="/opt/rook.github.io"
+
+# install node dependencies through container
+echo "pulling npm dependencies..."
+docker run -t --user 1000 -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine npm install
```

Comment:
I might be missing something. docker run start a new container, and the subsequent docker run below starts a different container. I think you are trying to run two commands in a single container, right? you might need a wrapper script to do the npm install and node build/.

Maybe this is working now because node:8-alpine already as `nom install` done?